lisa/sut_orchestrator/azure/features.py

```diff
@@ -108,10 +121,126 @@ def _execute(self, wait: bool, operator: str, **kwargs: Any) -> None:
             wait_operation(operation, failure_identity="Start/Stop")
 
 
+class FixedSerialPortsOperations(SerialPortsOperations):  # type: ignore
```

Comment:
Add comments to explain why it's here, and it doesn't need the old code comments to get it simpler.